includes/sanitizers/class-amp-style-sanitizer.php

```diff
@@ -1439,6 +1441,7 @@ private function get_parsed_stylesheet( $stylesheet, $options = [] ) {
 		$cache_key   = null;
 		$cached      = true;
 		$cache_group = 'amp-parsed-stylesheet-v24'; // This should be bumped whenever the PHP-CSS-Parser is updated or parsed format is updated.
+		$use_cache   = wp_using_ext_object_cache() || ! $this->args['allow_transient_caching'];
```

Comment:
The name is misleading, both object cache and transient cache are caches in some way.
`$use_cache` implies that caching in general is disabled if this is `false`.
```suggestion
		$use_object_cache = wp_using_ext_object_cache() || ! $this->args['allow_transient_caching'];
```